apps/documentation/src/routes/documentation/routing/dynamic-routes.mdx

```diff
@@ -13,19 +13,25 @@ import Breadcrumbs, { Element } from '@/components/breadcrumbs'
 
 > 🚧 Only available for SSR apps. This feature will be soon available also for static generated apps.
 
-When you don't know the exact segment names ahead of time and
-want to create routes from dynamic data, you can use
-Dynamic Segments that are filled in at request time.
+When the exact segment names are not known in advance and you need to generate routes from dynamic data,
+you can use dynamic segments, which are populated at request time.
 
 ## Convention
 
-A Dynamic Segment can be created by wrapping a
-file or folder name in square brackets: `[segmentName]`. For example, `[id]` or `[slug]`.
+A dynamic segment can be created by wrapping a
+file or a folder name in square brackets: `[segmentName]`.
 
-## Example
+You are going to probably use something like:
```

Comment:
I'm not a fan of "personalizing" the documentation by pointing directly the reader. 
I'll think about a rephrase over this sentence